bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx70-de.html

```diff
@@ -100,7 +100,7 @@ <h3 class="c-picto-block-title">{{ _('Schütze deine Passwörter und nimm sie mi
 
   <aside class="mzp-l-content c-utilities">
     <p>
-    {% trans notes=url('firefox.notes') %}
+    {% trans notes=url('firefox.releases.index') if not version else '/firefox/%s/releasenotes/'|format(version) %}
```

Comment:
Note for reviewer: The URL `/firefox/74.0beta/releasenotes/` will 404 currently, but that's because those release notes have not yet been published.